cpg-language-llvm/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/llvm/LLVMIRLanguageFrontendTest.kt

```diff
@@ -51,6 +51,15 @@ class LLVMIRLanguageFrontendTest {
         frontend.parse(topLevel.resolve("main.ll").toFile())
     }
 
+    @Test
+    fun testVectorPoison() {
+        val topLevel = Path.of("src", "test", "resources", "llvm")
+
+        val frontend =
+            LLVMIRLanguageFrontend(TranslationConfiguration.builder().build(), ScopeManager())
+        frontend.parse(topLevel.resolve("vector_poison.ll").toFile())
+    }
```

Comment:
Maybe a quick assert here would be helpful to understand how the graph should look like